scripts/next-data/generatePreBuildFiles.mjs

```diff
@@ -43,16 +67,20 @@ export const generateWebsiteFeeds = cachedBlogData =>
       ? `/en/blog/${metadata.blogCategory}`
       : '/en/blog';
 
-    const mapBlogPostToFeed = post =>
+    const mapBlogPostToFeed = async post => {
+      const markdownContent = await readFile(join(blogPath, post.category, post.file), 'utf8');
```

Comment:
We're over-engineering things here. The whole already rendered blog post should be provided directly from here https://github.com/nodejs/nodejs.org/blob/main/next.data.mjs#L16.

Nextra already gives us the whole tree containing the full grey-matter, frontmatter and markdown content + parsed markdown content, afaik.

If it doesn't provide the parsed markdown into HTML, then at least we don't need to re-read the file with `readFile` here 👀 

Also we don't need to use `async` and `await` here... We can just use plain promises. `renderMarkdown.then`